General/Settings.md

```diff
@@ -0,0 +1,292 @@
+# Settings
+
+A Pharo developer can define **settings** to give users a way to easily customise... well, anything really.
+The advantages of using the settings framework of Pharo are:
+- Users can change the value of any setting defined in the image with the **Settings Browser** window. No need to build a custom UI for it.
+- It is easy to add settings to an already existing application. A setting is basically just a class variable (or an instance variable of a singleton object) whose value can be set from the Settings Browser.
+- No dependency. Your application does not depend on the **Pharo Settings Framework**, and the Pharo Settings Framework does not depend on your application.
+
+## Table of Contents
+
+* [The Settings Browser](#the-settings-browser)
+* [Declaring a Setting](#declaring-a-setting)
+	+ [As quick as possible](#as-quick-as-possible)
+	+ [WARNING!: Default value of a setting](#warning---default-value-of-a-setting)
+	+ [Elements of a Setting Declaration](#elements-of-a-setting-declaration)
+		- [Id of the setting (#mySetting)](#id-of-the-setting---mysetting-)
+		- [Is it a setting or a group node? (setting:)](#is-it-a-setting-or-a-group-node---setting--)
+		- [Where is the variable this setting affects? (target:)](#where-is-the-variable-this-setting-affects---target--)
+		- [Label and Description (label: description:)](#label-and-description--label--description--)
+		- [Parent (parent:)](#parent--parent--)
```

Comment:
Those links are not working, I guess because of the extra dashes in the references.